tests/integration/core/remote_operations.py

```diff
@@ -859,7 +861,7 @@ def get_pacemaker_targets(self, server, running=False):
         crm_resources = result.stdout.split('\n')
         targets = []
         for r in crm_resources:
-            if not re.search('chroma:Target', r):
+            if not re.search('lustre:Lustre', r):
```

Comment:
Does this need to cover ZFS as well?

Reply:
No, we only care about the main lustre resource.